styles/mixins/_layout.scss

```
@@ -48,39 +47,24 @@ $-color-overlay: var.$color-overlay;
 }
 
 /**
- * Establish a vertical (margin) rhythm for this element's immediate
- * children (i.e. put equal space between children).
+ * Put `$size` of vertical space between all immediate children.
  *
- * This mixin uses `!important` such that it can compete with specificity
- * of reset rules that set some of our element's margins to 0. That allows
- * these rules—which are applied to a parent element—to be able to assert
- * margins, as it should be able to do.
- *
- * This mixin uses a "lobotomized owl" selector.
- * See https://css-tricks.com/lobotomized-owls/
- *
- * @param $size [$-space]: Spacing size (padding)
+ * @param {length} [$size] - Relative size of spacing, 0 - 9. Default 5
```

```suggestion
 * @param {number} [$size] - Relative size of spacing, 0 - 9. Default 5
```

To me it's clear from the description that this is a number range. Unless there is a specific reason to use "length" which I'm not sure is standard?

---

I am sort of forcing a round peg into a square hole here, but I was trying to be explicit that the "type" (I can't put enough quotation marks around that here) should be a CSS `length`. https://developer.mozilla.org/en-US/docs/Web/CSS/length